pkg/controllers/state/statenode.go

```diff
@@ -296,7 +296,11 @@ func (in *StateNode) VolumeLimits() scheduling.VolumeCount {
 }
 
 func (in *StateNode) PodRequests() v1.ResourceList {
-	return resources.Merge(lo.Values(in.podRequests)...)
+	var totalRequests v1.ResourceList
+	for _, requests := range in.podRequests {
```

Comment:
`lo.Reduce`?

Reply:
I'm not a huge fan of lo and it would require more benchmarking to see if that hurts performance.